csrc/mmdeploy/preprocess/transform/normalize.cpp

```diff
@@ -99,13 +99,14 @@ class Normalize : public Transform {
       Tensor dst;
       if (to_float_) {
         OUTCOME_TRY(normalize_.Apply(tensor, dst));
+        data[key] = std::move(dst);
```

Comment:
There is already `data[key] = std::move(dst);` after the if-else condition branch